### ISSUE TITLE
Content-Type header not recognized in vsc.utils.rest_oauth

### ISSUE THREAD

**Issue by backelj** [hpcugent/vsc-utils#62]
Hi,

I was checking the function request_access_token from vsc.utils.rest_oauth, but it fails to work because of the line:

`request.add_header('Content-Type', 'application/json')`

It looks like this has to be written without the hyphen:

`request.add_header('ContentType', 'application/json')`

Can someone confirm this?

-- Thanks,

Franky

**Comment by wpoely86** [comment]
That seems odd. It should have a hyphen: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Type

**Comment by backelj** [comment]
@wpoely86 You're right, but the code as it is now, does not seem to work. But that's probably due to some other issue (which I haven't found just yet). In fact, commenting out that one line in request_access_token makes it work again.

**Comment by stdweird** [comment]
@backelj where do you get an error?

**Comment by wpoely86** [comment]
This `request_access_token` should probably be moved to vsc-accountpage-clients. It looks like something from long ago?

**Comment by backelj** [comment]
@stdweird I was just calling request_access_token from a test script... The error I get is: 

 File "[...]/vsc/utils/rest_oauth.py", line 45, in request_access_token
    uri = opener.open(request)
  File "/usr/lib64/python2.7/urllib2.py", line 437, in open
    response = meth(req, response)
  File "/usr/lib64/python2.7/urllib2.py", line 550, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib64/python2.7/urllib2.py", line 475, in error
    return self._call_chain(*args)
  File "/usr/lib64/python2.7/urllib2.py", line 409, in _call_chain
    result = func(*args)
  File "/usr/lib64/python2.7/urllib2.py", line 558, in http_error_default
    raise HTTPError(req.get_full_url(), code, msg, hdrs, fp)
urllib2.HTTPError: HTTP Error 400: Bad Request

As said, commenting out the Content-Type line in request_access_token makes it work.

**Comment by stdweird** [comment]
ok, looks like an error. afaik, we always use curl to fetch a new token and do not specify the content type, so i guess the json content type is not supported with this url. feel free to make a PR to remove the line
